lib/repl.js

```diff
@@ -1566,8 +1567,10 @@ function defineDefaultCommands(repl) {
     help: 'Print this help message',
     action: function() {
       const names = ArrayPrototypeSort(ObjectKeys(this.commands));
-      const longestNameLength = MathMax(
-        ...ArrayPrototypeMap(names, (name) => name.length)
+      const longestNameLength = ArrayPrototypeReduce(
+        names,
+        (max, name) => MathMax(max, name.length),
+        0
```

Comment:
If we wait for https://github.com/nodejs/node/pull/37005 to land, we could use `MathMaxApply` and get better perf results.